core/src/wsv.rs

```diff
@@ -173,6 +173,20 @@ impl<W: WorldTrait> WorldStateView<W> {
             .collect()
     }
 
+    /// Calculates network TPS.
+    ///
+    /// `None` means that transactions can not yet be calculated.
+    #[allow(clippy::float_arithmetic, clippy::cast_precision_loss)]
```

Comment:
Or we could just use the conveniently provided FixNum crate. We lose nothing, since both time and number of transactions is computed as `u64`. I don't see why we can't return that tuple either, and offload actually computing the TPS to Prometheus, or the data scientist working on it.

Reply:
We have `fixnum` for currency related calculations mostly. For TPS I don't think precision matters that much. What do you mean by raw result?

Reply:
I meant `(time_duration, number_of_transactions)`, i.e. not doing any server-side computations. This should take care of the `floating_point_arithmetic` warnings.